editor/src/components/editor/actions/action-utils.ts

```diff
@@ -362,3 +364,74 @@ export function simpleStringifyActions(
     .map((a) => simpleStringifyAction(a, indentation))
     .join(`,\n${spacing}`)}\n${spacingBeforeClose}]`
 }
+
+export function getElementsToNormalizeFromCommands(commands: CanvasCommand[]): ElementPath[] {
+  return mapDropNulls((command) => {
+    switch (command.type) {
```

Comment:
Minor, but this being exhaustive ensures we have to fix it when adding a new `CanvasCommand`.